docs/safemode.md

```diff
@@ -0,0 +1,58 @@
+# Safemode
+
+Safemode represents a number of safety nets we have implemented into the chaos controller to help new and experienced users feel more confident deploying new disruptions to their environments.
+
+The Chaos Controller can be scary to use in production environments, but we learn a lot more running chaos experiments in our production environments. Considering this, having safety nets in place makes the entire process of running chaos experiments in high value environments a little safer.
+
+The disruption config has a parameter appropriately named `safeMode` which represents a spec containing several boolean fields.
+One of the fields, `IgnoreAll` overrides the rest if set to true. It will ignore every safety net.
+We recommend that `IgnoreAll` be set on the cluster level by default, so maintainers of the controller can turn it on by default in production and maybe off in staging environments.
```

Comment:
Sorry that I'm getting so easily confused by double negatives, but we have a field `IgnoreAll`. We recommend it be ON (so true?) in prod. When set to true, we ignore all safety nets? Isn't that the opposite of what we want in prod?

Reply:
IgnoreAll would be set to false in prod. In our global configurations, we would have ignoreAll set automatically depending on environment, so in prod it would be false. So we recommend it to be False in prod.

Reply:
> maintainers of the controller can turn it on by default in production

> IgnoreAll would be set to false in prod

So the comment is backwards?

Reply:
Changing this format in the next commit, so resolving this conversation in favor for the new `Unsafemode` schema.